web/jinja2/index.html

```diff
@@ -1,9 +1,141 @@
 
 {% extends '_base.html' %}
 {% block main_content %}
-    <ul>
-        {% for team in teams %}
-            <li><a href="{{ url('team_detail', args=[team.pk]) }}">{{ team.name }}</a></li>
-        {% endfor %}
-    </ul>
+    <div class="col-sm-12">
```

Comment:
May we should split this into different html files with each section. What do you think?